src/vs/workbench/parts/extensions/browser/extensionsList.ts

```diff
@@ -165,7 +165,7 @@ export class Renderer implements IPagedRenderer<IExtension, ITemplateData> {
 
 		const extRecommendations = this.extensionTipsService.getAllRecommendationsWithReason();
 		if (extRecommendations[extension.id.toLowerCase()]) {
-			data.root.setAttribute('aria-label', extension.displayName + '. ' + extRecommendations[extension.id]);
+			data.root.setAttribute('aria-label', extension.displayName + '. ' + extRecommendations[extension.id].reasonText);
```

Comment:
we should probably add a `.toLowerCase()` for the `extension.id`. That is what the `if` looks for in line 168